app/src/main/java/pl/futurecollars/invoicing/db/DatabaseConfiguration.java

```diff
@@ -0,0 +1,36 @@
+package pl.futurecollars.invoicing.db;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import pl.futurecollars.invoicing.db.file.IdCurrentNumber;
+import pl.futurecollars.invoicing.db.files.FileDatabase;
```

Comment:
pl.futurecollars.invoicing.db.files.FileDatabase;
ścieżka powinna być db.file, bez s

Reply:
wymagało to operacji zmiany nazwy katalogu z file -> files -> file,  ale zadziałało :)